src/services/CleanupService.ts

```diff
@@ -443,15 +467,25 @@ export const clearSiteDataForThisDomain = async (
 };
 
 export const removeSiteData = async (
-  siteData: string,
-  isChrome: boolean,
+  state: State,
+  siteData: SiteDataType,
+  browserDetect: string,
   domains: string[],
   debug: boolean,
 ): Promise<boolean> => {
-  const listName = isChrome ? 'origins' : 'hostnames';
+  const listName = ((b: string) => {
+    switch (b) {
+      case 'Chrome':
```

Comment:
Optional for this PR. This can be an enum.

Reply:
Sure.  As long as we don't modify the return string from browserDetect.js in global_files

Reply:
Resolved through 17b7bff